core/bin/external_node/src/config/observability.rs

```diff
@@ -1,43 +1,99 @@
-use zksync_config::configs::ObservabilityConfig;
-
-pub fn observability_config_from_env() -> anyhow::Result<ObservabilityConfig> {
-    // The logic in this method mimics the historical logic of loading observability options
-    // This is left intact, since some of the existing deployments may rely on the this behavior.
-    let sentry_url = if let Ok(sentry_url) = std::env::var("MISC_SENTRY_URL") {
-        if sentry_url == "unset" {
```

Comment:
Is it reasonable to remove `unset` logic? Maybe I'm missing something obvious, but it seems that not having the corresponding env variable set would suffice.

Reply:
I think people (myself included) are afraid to remove it to not accidentally put something in crashloop if some ancient component has `unset` in its configmap 😅
So to remove it, we must ensure that nobody (including external parties) has it in configmap.

Reply:
Returned it back just to be safe.

Reply:
We have some `unset`s in [infra](https://github.com/matter-labs/gitops-kubernetes/blob/main/apps/common/zksync-v2-config/zksync-v2-config.yaml#L76)